### ISSUE TITLE
[Bug]: No Output when an Exception occurs

### ISSUE THREAD

**Issue by marclangesauels** [Roboroads/laravel-tinker#320]
### Short bug description

No Output when an Exception occurs

### Extended bug description

When the code you try to run produces an error no output is shown. This was not the case with the prevoius version (2.6.1)
![Screenshot 2024-04-11 134959](https://github.com/Roboroads/laravel-tinker/assets/67956769/21b08edc-e19d-450b-8c2f-afb669de5162)

### Your project is not the problem

- [X] I tried using a fresh laravel install - I'm fairly certain it has nothing to do with my project.

### How to reproduce?

tried provoking an error

### Expected behavior

showing an exception message

### Screenshots

![image](https://github.com/Roboroads/laravel-tinker/assets/67956769/d6dd23bc-ccd5-48af-83d4-25a0ccc30ffe)

### IDE & Version

2024.1

### PHP Version

8.2

### OS and version

Windows 11

### Laravel Tinker plugin version

2.7.0

### Interpreter type

Docker compose

### Laravel Version

9.52.16

### Tinker version

2.9.0

### Psysh version

0.12.2

**Comment by pindab0ter** [comment]
Any update on this?

**Comment by N-Molham** [comment]
Thanks @Roboroads for your efforts.
I am also waiting on a fix for this, which also I assume it will also fixes the #326 issue.

**Comment by Roboroads** [comment]
> I assume it will also fixes the #326 issue.

It will, yea. Sorry it's taking so long. I think the main guy that helps with PHP is on holiday.

**Comment by N-Molham** [comment]
No worries @Roboroads, no need for apology. Appreciate the response and your efforts put into this plugin. It is a game changer for me

**Comment by pindab0ter** [comment]
Agreed, we really appreciate the effort you're putting into this! The extension has been so great for me. Beats going into the CLI and setting up your environment each time or using a dedicated tinkering route. I love being able to whip up a tinker session with a shortcut and just trying/verifying stuff.

We'd love for it to be fixed, but no need to apologise. I've had PRs merged four years after I've opened them barely any communication, so I'm used to much worse, haha.

**Comment by mvdgeijn** [comment]
I'm getting weird output when tinkering:

```php
print "Test";
```

Same issue?

**Comment by Roboroads** [comment]
@mvdgeijn yes, it is. I'm seriously considering rolling back until I get a response from JetBrains.

**Comment by zaalbarxx** [comment]
Not sure if it is related, @Roboroads, but for me the output will always be empty when exception was thrown somewhere, like here:
![image](https://github.com/Roboroads/laravel-tinker/assets/3301242/b7f450aa-dba5-4e59-bd76-25f7cf544149)

**Comment by pindab0ter** [comment]
Until this is resolved, this is what I'm using in my Tinker Console to get by:

```php

try {
    $result = 'Foo';
    
    dump($result);
} catch(Exception $e) {
    dump($e->getMessage());
    dump($e->getTraceAsString());
}
```

**Comment by zaalbarxx** [comment]
Yeah, that's what I did :) Just making sure that the kind of issue I encountered is the same as the one from this isue :) Thanks anyway :) Cheers!

**Comment by Roboroads** [comment]
Little update, JB has responded with "will take a look at it" - which has been 2 weeks ago. I'm getting to a point where I just want to rollback to 2.6 and then have people on windows deal with their Arabic/Cyrillic text issues for a while again..

**Comment by shina** [comment]
As a mac user, I'm up to it 😅

**Comment by pindab0ter** [comment]
I think it would be a good idea to make that decision based on the size of the impacted groups, for as far as you are able to determine that.

**Comment by Roboroads** [comment]
Hi all. I decided to revert back to 2.6 way of handling output. Release is pending into 2.7.1 and will be available for download shortly. This does mean that I will have to reopen #255 for the time being. Sorry windows users husing arabic/cyrillic text :(

**Comment by pindab0ter** [comment]
Just got the update, I'm so happy that it's working as intended again! Thank you so much! 🙏

**Comment by Roboroads** [comment]
Oh yea, forgot to update this. I figured it out :) It's fixed now ^^
